services/QuillLMS/app/serializers/plan_serializer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: plans
+#
+#  id              :bigint           not null, primary key
+#  audience        :string           not null
+#  display_name    :string           not null
+#  interval        :string
+#  interval_count  :integer
+#  name            :string           not null
+#  price           :integer          default(0)
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  stripe_price_id :string
+#
+# Indexes
+#
+#  index_plans_on_name  (name) UNIQUE
+#
+class PlanSerializer < ActiveModel::Serializer
+  attributes :display_name, :price_in_dollars, :stripe_price_id
+
+  def price_in_dollars
+    object.price / 100
```

Comment:
Price is an integer, let's make the denominator a float to avoid weird integer division.
```suggestion
    object.price / 100.0
```

<img width="350" alt="Screen Shot 2022-04-07 at 5 15 44 PM" src="https://user-images.githubusercontent.com/1304933/162314386-ac295a0d-b1eb-4870-823f-098af64ad964.png">